pom.xml

```diff
@@ -66,6 +66,12 @@
             <artifactId>spring-security-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
+        </dependency>
```

Comment:
This dependency is redundant. Spring Boot has this dependency in logging-starter, that is contained in spring-boot-starter.

Reply:
Done